src/components/Dashboard/TodoList.vue

```diff
@@ -1424,6 +1441,54 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
     }
   }
 
+  async loadContinuationInApplication (task: ApiTaskIF): Promise<void> {
+    const filing = task.task.filing
+    const header = filing.header
+    const continuationIn = filing.continuationIn
+
+    if (header) {
+      let subtitle: string = null
+      if (this.isStatusDraft(header)) {
+        if (this.getNameRequest) {
+          subtitle = `NR APPROVED - ${this.expiresText(this.getNameRequest)}`
+        } else {
+          subtitle = 'DRAFT'
+        }
+      }
+
+      const paymentStatusCode = header.paymentStatusCode
+      const payErrorObj = paymentStatusCode && await PayServices.getPayErrorObj(this.getPayApiUrl, paymentStatusCode)
+
+      // NB: continuationIn application may be undefined
+      const haveData = Boolean(
+        continuationIn?.offices ||
+        continuationIn?.contactPoint ||
+        continuationIn?.parties ||
+        continuationIn?.shareClasses
+      )
```

Comment:
There are a couple of new object that should add in here (still undefined - see Teams for discussion).

Reply:
how about add comment here to add those after we identify all?

Reply:
We might not remember those comments. Like the comment below, how about adding a comment to the "Cont In ledger" ticket to come back here and add the missing objects?

Reply:
Makes sense. Will do. Is this the correct ticket? https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/20808 ?

Reply:
Yes, that's the one.

Reply:
Added a point in To Do section in the ticket [20808](ttps://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/20808)